app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
+  def create
```

Comment:
- [ ] Please make sure that you have the `new` method which is responsible for instantiating a new Comment object without saving it to the database. This new comment will be used to render the form in your `views/comments/new.html.erb`.  Please note that you should have the form for commenting on a post in `views/comments/new.html.erb` with the route `GET /posts/:post_id/comments/new` :pray: